Split liquid_viscosity to liquid_viscosity and move_resistance (#10810)
minetest/minetest: doc/lua_api.txt

        climbable = false,  -- If true, can be climbed on (ladder)

        move_resistance = 0,
        -- Slows down movement of players through this node (max. 7).

>         -- If this is nil, it will be equal to liquid_viscosity.

Reviewer: Sounds a bit weird.
```suggestion
        -- If nil or not specified, the value will default to the "liquid_viscosity".
```

Reviewer 2: Sounds absolutely correct to a Lua developer. Unspecified and nil are pretty much the same thing (except for varargs, but that's largely irrelevant, this is about tables).
